package.json

```diff
@@ -24,17 +24,16 @@
   },
   "dependencies": {
     "graceful-fs": "^4.1.11",
-    "through2": "^2.0.3"
+    "streamx": "^2.6.0"
   },
   "devDependencies": {
     "eslint": "^6.8.0",
-    "eslint-config-gulp": "^4.0.0",
-    "expect": "^25.4.0",
-    "jest-mock": "^25.5.0",
-    "mississippi": "^1.3.0",
+    "eslint-config-gulp": "^5.0.0",
+    "expect": "^26.0.1",
```

Comment:
@coreyfarrell I've been thinking a lot about providing a "prelude" (been writing a lot of Rust) for gulp testing and publishing as `@gulpjs/test` then using it in all of the projects. I bring it up because jest just bumped versions to a major again, which means my scaffold is already behind 😭 

Do you think that's a good idea or should I defer it for now? Ideally this would also help plugins test better too, but that's definitely a future thing.

Reply:
TBH I'm not sure, I haven't gotten a chance to look very deep into the scaffold yet.  In general I use tap (and more recently libtap) to do my testing so for mocha testing I have to defer to what others think.

Reply:
🤔 We definitely have a unique setup with mocha but using the jest assertion stuff. I don't like the auto-mocking in jest so we didn't switch over to it completely. It also doesn't support forcing async tests, which is something I loved about `lab`.

I won't hang this PR on these ideas, just thought I'd ask your opinion. Sent a few more comments in a review though.